web/static/frontend/js/machine_page.js

```diff
@@ -4,14 +4,20 @@
 
 function machine_page() {
   document.addEventListener('DOMContentLoaded', () => {
-    const buttons = document.getElementsByClassName('machine-delete-button');
+    const delete_buttons = document.getElementsByClassName('machine-delete-button');
 
-    for (const button of buttons) {
+    for (const button of delete_buttons) {
       button.onclick = () => {
         return confirm('Are you sure you wish to delete this machine? You will lose all XP associated with it.');
       };
     }
 
+    const deactivate_buttons = document.getElementsByClassName('machine-deactivate-button');
+
+    for (const button of deactivate_buttons) {
```

Comment:
Identifier 'deactivate_buttons' is not in camel case.

Enforce the usage of camel case to have a consistent codebase.

[Read more about it here.](http://eslint.org/docs/rules/camelcase)